src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/Table.tsx

```diff
@@ -115,52 +115,34 @@ function attachTreeHandler(cell: TableCommonCell, tableProps: TableProps) {
 export class Table extends React.PureComponent<TableProps, TableState> {
     state: TableState = {waitingForFont: true, dataTableRefInitialized: false};
 
-    private timer: ReturnType<typeof setTimeout> | null = null;
     private dataTableRef?: ChartKitDataTable;
     private id?: string;
 
     componentDidMount() {
         validateConfigAndData({data: this.props.data.data, config: this.props.data.config});
-        // @ts-ignore | ts doesn't know about FontFaceSet API
-        document.fonts.load('700 10pt "YS Text"').then((fonts) => {
-            if (!fonts.length) {
-                console.warn(
-                    'The font used for the table headers has not loaded,' +
-                        ' the table may not be displayed correctly',
-                );
-            }
 
+        // fix same as for table at CHARTS-7640
```

Comment:
It's much better if you describe case instead of add link to closed for external contributors ticket

Reply:
Added additional commentary